packages/sit-onyx/src/components/OnyxMiniSearch/OnyxMiniSearch.vue

```diff
@@ -45,13 +45,18 @@ defineExpose({
 </script>
 
 <template>
-  <div :class="['onyx-mini-search', densityClass]" v-bind="rootAttrs">
+  <div
+    :class="['onyx-mini-search', densityClass]"
+    v-bind="rootAttrs"
+    :style="{ '--onyx-placeholder-character-count': input?.placeholder.length }"
```

Comment:
```suggestion
    :style="{ '--onyx-placeholder-character-count': input?.placeholder.length }"
```

Using the placeholder like this is not reactive so it will not update if the language changes. Would prefer to create a computed for the placeholder text and use it here + as input placeholder